template/config/meta/.babelrc

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
This file must be part of the storybook module: not the core config module.

Reply:
fixed